<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Alans-MacBook-Air.local" failures="3" tests="10" name="Test" time="0.948" errors="0" timestamp="2024-07-16T16:51:33 EDT">
  <testcase classname="com.finalproject.loanAPITests" name="postReqUserAuth" time="0.597"/>
  <testcase classname="com.finalproject.loanAPITests" name="getCustomerInfo" time="0.022"/>
  <testcase classname="com.finalproject.loanAPITests" name="getCreditScore" time="0.029"/>
  <testcase classname="com.finalproject.loanAPITests" name="getLoanEligibility" time="0.02"/>
  <testcase classname="com.finalproject.loanAPITests" name="getRiskAssessmentReport" time="0.025"/>
  <testcase classname="com.finalproject.loanAPITests" name="submitLoanApplication" time="0.045"/>
  <testcase classname="com.finalproject.loanAPITests" name="approveLoanApplication" time="0.013"/>
  <testcase classname="com.finalproject.loanAPITests" name="rejectLoanApplication" time="0.015">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path code doesn&amp;apos;t match.
Expected: &amp;lt;101&amp;gt;
  Actual: &amp;lt;211&amp;gt;
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path code doesn't match.
Expected: <101>
  Actual: <211>

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:270)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:117)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at com.finalproject.loanAPITests.rejectLoanApplication(loanAPITests.java:243)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- rejectLoanApplication -->
  <testcase classname="com.finalproject.loanAPITests" name="pullLoanApplicationStatus" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at com.finalproject.loanAPITests.pullLoanApplicationStatus(loanAPITests.java:254)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- pullLoanApplicationStatus -->
  <testcase classname="com.finalproject.loanAPITests" name="deleteLoanApplication" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at com.finalproject.loanAPITests.deleteLoanApplication(loanAPITests.java:281)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- deleteLoanApplication -->
</testsuite> <!-- Test -->
